js/src/mpl.js

```diff
@@ -160,7 +157,7 @@ mpl.figure.prototype._init_canvas = function() {
 
     var pass_mouse_events = true;
 
-    canvas_div.resizable({
+    /*canvas_div.resizable({
```

Comment:
Maybe in the `@jupyterlab/domutils` package I'm working on... https://github.com/blink1073/domutils